lang/cs/Org.Apache.REEF.Client/API/JobParameters.cs

```diff
@@ -81,6 +86,14 @@ public int DriverMemoryInMB
         }
 
         /// <summary>
+        /// The job submission environment variable map.
+        /// </summary>
+        public IDictionary<string, string> JobSubmissionEnvMap
+        {
+            get { return _jobSubmissionEnvMap; }
```

Comment:
This returns a reference to a mutable collection, which seems dangerous. Wouldn't it be better to return a copy? Or rather: Is access to this collection as a whole needed at all?

Reply:
Ok